### DISCUSSION TITLE
How can I help? I really like your project :D

### DISCUSSION THREAD

**Discussion by 404answernotfound** [alineacms/alinea#269] [Q&A]
Hello, I'd like to participate in the development of this CMS. Can I help you out? 🍡

**Comment by dmerckx** [comment]
Hey @404answernotfound, after an initial big developed sprint we shared the product and took a step back to gather some feedback. Right now we are using those learning to reevaluate our milestones and we are working on some new wireframes to kickstart another big development cycle. Feel free to share your thoughts about a revised content workflow here: https://github.com/orgs/alineacms/discussions/303

We also took notion of the paradigm shift towards React server components (by nextjs and other frameworks) and we are figuring out how we can best support this with Alinea and take full advantage of the added possibilities. Do you have any opions on that matter?
